docs/docs/user-manuals/how-to-use-frontend.md

```diff
@@ -0,0 +1,237 @@
+
+- Keycloak administrator
+- Online store administrator
+- Online store customer
+
+### User Management
+
+1. At the beginning, there are no users in the system. Consequently, when we navigate to the online store, we are not logged in as a user and in the top right corner it says "LOGIN".
+2. Clicking on "LOGIN" will redirect us to Keycloak. Here we have two options: We could try to log in with an existing user via username and password — this is impossible because there are no users yet — or we register a new user. To do this, we click on the "Register" link at the bottom of the dialog.
+3. We fill out the registration form and submit it by pressing the "Register" button.
+4. Keycloak redirects us or returns us to the online store. Now we are logged in.
+5. By default, we only have the permissions of a customer. That means we still can't do anything because there are no products in the online store yet.
+6. Therefore, we need to add additional permissions to our new user so that we can manage the online store. To do this, we navigate to Keycloak: http://localhost:4000/keycloak/
+7. We click on "Administration Console" and log in with the username "admin" and password "admin".
+8. In the top left corner, there is a dropdown menu. There you can select the so-called _realm_. We switch to "Misarch".
+9. Then we navigate to "Users" in the side menu. Now, the URL should look like this: http://localhost:4000/keycloak/admin/master/console/#/Misarch/users
+10. The user we created should now be listed under Users.
+11. We click on the user or its username.
+12. This takes us to the "User details".
+13. We go to the "Role mapping" tab.
+14. We click on the blue "Assign role" button. A dialog opens.
+15. We check the "admin" and "employee" roles and confirm the dialog with "Assign".
+16. Now the user has all available roles in the system, so the user can both administer the online store and shop in the online store.
+17. We navigate back to the online store. We should still be logged in. The view has changed because now we are logged in not as a customer but as a store employee or administrator. If not, just reload the page.
+18. Now the main menu is on the right instead of the left side, and it points to pages that allow store management.
+19. Very important! If we wanted to change the current user role, we could do so. In the top right corner, there is a button labeled "SWITCH USER ROLE". Clicking on it opens a dialog.
+20. In the dropdown, we can select one of the available roles. The selection takes effect immediately, and we must explicitly close the dialog by clicking "CANCEL".
+21. Clicking "LOGOUT" logs out the currently logged-in user.
+
+### Managing the Online Store
+
+To manage the online store, we need to log in with a user who has either the "admin" or "employee" role, or even both. In the previous section, we created such a user using Keycloak.
```

Comment:
IIRC, I set it up so that `admin != employee`, so especially `admin > employee` is **not** true when it comes to permissions.

Reply:
```suggestion
To manage the online store, we need to log in with a user who has the `employee` role. In the previous section, we created such a user using Keycloak.
Preferably, our user should even have `admin` capabilities to manage further configurations.
```